### ISSUE TITLE
No logs appearing, stuck in "waiting for runtime events" with create react app

### ISSUE THREAD

**Issue by TomYeoman** [wallabyjs/console-ninja#46]
The extension doesn't move from "waiting for runtime events", it seems to detect my create-react-app starting, via `yarn start`, I then navigate to http://localhost:3000 whereby I can view the console logs in chrome debugger, however not in editor.

I am running VSCode + the webpack server under Ubuntu 20.04.4 LTS on windows WSL2.

Logs are;

```
info extension stopping host
info host ------------ starting ------------
info host worker Starting Host Worker...
info extension host process output: port:[41011]
info extension 
ERR! extension error while searching files: {"message":"spawn /home/wsl-tom/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/node_modules.asar.unpacked/@vscode/ripgrep/bin/rg ENOENT","code":5}
info host client connected: extension { address: '::', family: 'IPv6', port: 41011 }
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/server.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/common/db_entities/ServerStats.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/PhaserEntityRenderer.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/bot/stressBot.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.32/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/server.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/common/db_entities/ServerStats.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/PhaserEntityRenderer.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/bot/stressBot.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host found 6 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host starting tools discovery and integration
info host looking for tools to patch
info host found 6 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info buildHook-2457 installing build hook for webpack at /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend
info buildHook-2457 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-2457 running tools: webpack
info buildHook-2457 installing fs interceptor
info host client connected: buildHook { address: '::', family: 'IPv6', port: 41011 }
info buildHook-2457 file processed (stat): 5 log points, error handler: false
info buildHook-2457 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-2457 running tools: webpack
info buildHook-2457 file processed (stat): 0 log points, error handler: true
info buildHook-2457 file processed (stat): 1 log points, error handler: false
info buildHook-2457 file processed (stat): 0 log points, error handler: false
info buildHook-2457 file processed (stat): 1 log points, error handler: false
info host client disconnected: buildHook 4100
info host client disconnected: buildHook 4100, has clients false
info buildHook-3136 installing build hook for webpack at /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend
info buildHook-3136 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-3136 running tools: webpack
info host client connected: buildHook { address: '::', family: 'IPv6', port: 41011 }
info buildHook-3136 installing fs interceptor
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 5 log points, error handler: false
info buildHook-3136 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-3136 running tools: webpack
info buildHook-3136 file processed (stat): 0 log points, error handler: true
info buildHook-3136 file processed (stat): 0 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 0 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info buildHook-3136 file processed (stat): 0 log points, error handler: false
info buildHook-3136 file processed (stat): 1 log points, error handler: false
info extension stopping host
info host ------------ starting ------------
info host worker Starting Host Worker...
info extension host process output: port:[42547]
info extension 
ERR! extension error while searching files: {"message":"spawn /home/wsl-tom/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/node_modules.asar.unpacked/@vscode/ripgrep/bin/rg ENOENT","code":5}
info host client connected: extension { address: '::', family: 'IPv6', port: 42547 }
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/server.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/common/db_entities/ServerStats.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/game/index.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/App.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/features/toolbar/Toolbar.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/features/playerhud/GameStatusHUD.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/PhaserEntityRenderer.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/bot/stressBot.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 6 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/server.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/common/db_entities/ServerStats.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/game/index.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/App.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/features/toolbar/Toolbar.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/features/playerhud/GameStatusHUD.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/PhaserEntityRenderer.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/bot/stressBot.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 6 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info extension stopping host
info host ------------ starting ------------
info extension host process output: port:[37185]
info host worker Starting Host Worker...
info extension 
ERR! extension error while searching files: {"message":"spawn /home/wsl-tom/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/node_modules.asar.unpacked/@vscode/ripgrep/bin/rg ENOENT","code":5}
info host client connected: extension { address: '::', family: 'IPv6', port: 37185 }
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/systems/PlayerSystem.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host removing cache folder: e53e7a8f07311d646f5c85759e12e363
info extension stopping host
info host removing cache folder: 6df53f76995c65d3728fc3302d9d2d3e
info extension stopping host
info extension host process output: port:[40317]
info host ------------ starting ------------
info extension 
info host worker Starting Host Worker...
ERR! extension error while searching files: {"message":"spawn /home/wsl-tom/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/node_modules.asar.unpacked/@vscode/ripgrep/bin/rg ENOENT","code":5}
info host client connected: extension { address: '::', family: 'IPv6', port: 40317 }
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/server/src/game/main.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/server/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info extension stopping host
info host ------------ starting ------------
info host worker Starting Host Worker...
info extension host process output: port:[40583]
info extension 
ERR! extension error while searching files: {"message":"spawn /home/wsl-tom/.vscode-server/bin/6261075646f055b99068d3688932416f2346dd3b/node_modules.asar.unpacked/@vscode/ripgrep/bin/rg ENOENT","code":5}
info host client connected: extension { address: '::', family: 'IPv6', port: 40583 }
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/game/Main.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/Simulator.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host starting host: {"extensionFolder":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35","runtimeHookFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/tracer.js","globalErrorHandlerFile":"/home/wsl-tom/.vscode-server/extensions/wallabyjs.console-ninja-0.0.35/out/runtimeHook/errorHandler.js","filesToInstrument":["/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/scenes/level-one.ts","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/game/Main.tsx","/home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/src/Simulator.ts"],"debug":false,"config":{"dateTimeFormat":"hh:mm:ss.SSS","openLinksInEditor":true,"maxLogViewerEntries":15000,"autoClearLogViewerOnFileChanges":true,"outputMode":"Beside File","toolsToAutoPatch":["vite","jest","webpack","next.js","cypress","http-server","serve","hydrogen"]},"allowedHosts":["localhost","127.0.0.1","example.cypress.io","LAPTOP-3T3IEPN6","172.18.232.187"]}
info host starting tools discovery and integration
info host looking for tools to patch
info host found 3 tool file(s) to patch
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/webpack/lib/webpack.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest/bin/jest.js
info host patching /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend/node_modules/jest-runner/build/testWorker.js
info host completed tools discovery and integration
info host removing cache folder: 5392084ddb38d93fdc623c1f8c497866
info buildHook-7268 installing build hook for webpack at /home/wsl-tom/git/personal/zombie-io-game-matter-port/frontend
info buildHook-7268 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-7268 running tools: webpack
info buildHook-7268 installing fs interceptor
info host client connected: buildHook { address: '::', family: 'IPv6', port: 40583 }
info buildHook-7268 file processed (stat): 3 log points, error handler: false
info buildHook-7268 file processed (stat): 6 log points, error handler: false
info buildHook-7268 file processed (stat): 12 log points, error handler: false
info buildHook-7268 allowed tools: vite, jest, webpack, next.js, cypress, http-server, serve, hydrogen
info buildHook-7268 running tools: webpack
info buildHook-7268 file processed (stat): 0 log points, error handler: true

```

**Comment by smcenlly** [comment]
Console Ninja was not working for browser applications launched from `wsl` or `ssh` remote instances.

This has been fixed in Console Ninja `v0.0.37`. Please update to the latest version, it should work for you.

**Comment by TomYeoman** [comment]
Thanks for the quick fix, I can confirm it's now working :)
